op-batcher/batcher/channel_builder_test.go

```
@@ -810,3 +813,58 @@ func blockBatchRlpSize(t *testing.T, b *types.Block) int {
+	sbb := derive.NewSpanBatchBuilder(rollupCfg.Genesis.L2Time, rollupCfg.L2ChainID)
+	batch, l1Info, err := derive.BlockToSingularBatch(rollupCfg, l2block)
+	if err != nil {
+		return nil, err
+	}
+	sbb.AppendSingularBatch(batch, l1Info.SequenceNumber)
+	rawSpanBatch, err := sbb.GetRawSpanBatch()
+	if err != nil {
+		return nil, err
+	}
+	var buf bytes.Buffer
+	err = rlp.Encode(&buf, derive.NewBatchData(rawSpanBatch))
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)